hnix-store-core/src/System/Nix/DerivedPath.hs

```diff
@@ -25,12 +25,17 @@ import qualified System.Nix.StorePath
 
 data OutputsSpec =
     OutputsSpec_All
+  -- ^ Wildcard spec (!*) meaning all outputs
```

Comment:
```suggestion
  -- ^ Wildcard spec (^*) meaning all outputs
```
the syntax in the worker protocol is somewhat deprecated

Reply:
Fixed, thanks!